src/authenticator.ts

```diff
@@ -11,6 +11,10 @@ export interface TokenWithExpire {
   expire: number;
 }
```

Comment:
Could you update `TokenWithExpire` to `AccessTokenWithExpire`? That will help differentiate it from `Refresh` token.